website_sale_product_detail_attribute_value_image/templates/shop_product.xml

```diff
@@ -28,10 +28,7 @@
                 <t t-foreach="variant_id.value_ids.filtered(lambda x: x.website_product_detail_image_published)" t-as="attr_value">
                     <div class="flex-fill px-1">
                         <div class="text-center">
-                            <span
-                                t-field="attr_value.website_product_detail_image"
-                                t-options="{'widget': 'image', 'resize': '40x40'}"
-                            />
+                            <img class="img" t-attf-src="/web/image?model=product.attribute.value&amp;field=website_product_detail_image&amp;id=#{attr_value.id}" t-att-title="attr_value.website_name or attr_value.name" t-att-alt="attr_value.website_name or attr_value.name"/>
```

Comment:
Why the website name? The normal would be the attribute name itself as first option, IMO

Reply:
This follows the behavior of 'website_sale_product_attribute_image' which also includes a "website_name" field. Normal is to only use the "name" field, but if you want to change the way they are displayed on the website, you can use "website_name"; in this case, the value of this field wins.

Reply:
Sorry, you're right. It's ok!